migrations/20191004-add_parameters_as_mediumtext-pipelines.js

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable new-cap */
+
+'use strict';
+
+const prefix = process.env.DATASTORE_SEQUELIZE_PREFIX || '';
+const table = `${prefix}pipelines`;
+
+module.exports = {
```

Comment:
@parthasl Please take a look, as I'm not confident enough for DB migration

Reply:
👍